src/sessions/softphone-session-handler.ts

```diff
@@ -137,14 +137,17 @@ export default class SoftphoneSessionHandler extends BaseSessionHandler {
 
     this.checkForCallErrors(update, participant, callState);
 
+// TODO: we aren't getting 'handledPendingSession' if: we are on a call, then place an outbound call. That outbound does not emit 'handledPendingSession'
```

Comment:
Still need to iron out this TODO. It isn't necessarily blocking, but it really should work.